### DISCUSSION TITLE
Find my phone

### DISCUSSION THREAD

**Discussion by user84918342** [InfiniTimeOrg/InfiniTime#1834] [Q&A] [answered]
I don't have a device yet but I am still curious to know if it has some "Find my phone" functionality – I couldn't find anything at all concerning this (besides maybe using the music player but that doesn't sound convenient).

With "Find my phone" I don't mean anything advanced. For instance, a simple signal over Bluetooth to the phone to play an alarm would be enough.

**Comment by FintasticMan** [comment, marked as answer]
There isn't a feature like that yet, but we do want it, see #343. There was an implementation in #1193, but it wasn't done the way we want.
